src/clients/subgraph/queries/getAssetsInAccount/useGetAssetsInAccountSubgraph.ts

```diff
@@ -1,12 +1,12 @@
+import { request } from 'graphql-request';
 import { useQuery } from 'react-query';
 
 import FunctionKey from 'constants/functionKey';
-
-import subgraphClient from '../../subgraphClient';
+import { AssetsInAccountQueryDocument, subgraphEndpoint } from 'types/subgraph/gql/queries';
 
 const useGetAssetsInAccountSubgraph = (accountAddress: string) =>
   useQuery([FunctionKey.GET_ASSETS_IN_ACCOUNT, { accountAddress }], () =>
-    subgraphClient.AssetsInAccountQuery({ accountAddress }),
+    request(subgraphEndpoint, AssetsInAccountQueryDocument, { accountAddress }),
```

Comment:
In order to keep everything contained within modules, we could create a subgraph client just like before except using `graphql-request` and the generated types.
This way everything would be imported from one place (and mocked in tests when necessary).